src/icons/svgs/arrow_back_24px_rounded.svg

```diff
@@ -1,5 +1,5 @@
-<svg fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It's important not to specify the `fill` or `fill-opacity` in the svg so that it can be specified by the user of the icon.